docs/automerge.md

```diff
@@ -1,7 +1,7 @@
 ---
-title: "Automerge Pull Requests"
-description: "How to use labels and commands to automerge/autorebase your pull request."
-type: docs
+Title: "Automerge Pull Requests"
+Description: "How to use labels and commands to automerge/autorebase your pull request."
+Type: Docs
```

Comment:
I believe the capitalization here is a style preference, not a grammatical error.

Reply:
Thanks @jaellio for the review will close the PR